var/spack/repos/builtin/packages/recon/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from os import symlink
+
+
+class Recon(MakefilePackage):
+    """RECON: a package for automated de novo identification of repeat families
+       from genomic sequences."""
+
+    homepage = "http://eddylab.org/software/recon/"
+    url      = "http://eddylab.org/software/recon/RECON1.05.tar.gz"
+
+    version('1.05', sha256='4d4f76f439bcffd50380cffc41a80dc15fa4a80f38a04234e24da893ed7c025a')
+
+    variant('repeatmasker', default=False,
+            description='Use RepeatMasker developer patches (1.08)')
+
+    patch('repeatmasker_recon.patch', when='+repeatmasker')
```

Comment:
This looks great to me! Just to clarify, is this patch needed to build another piece of software that depends on it, or just to offer different functionality? Also, how does the version of RepeatMasker relate to the version of RECON?

Reply:
The patch just fixes a couple older bugs in RECON and is recommended by the RepeatMasker developers. I don't think it adds any new functionality to the software.